sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/properties/AadAuthenticationProperties.java

```diff
@@ -128,12 +137,19 @@ public class AadAuthenticationProperties implements InitializingBean {
     private Boolean sessionStateless = false;
 
     /**
-     * The OAuth2 authorization clients.
+     * The OAuth2 authorization clients, contains the authorization grant type, client authentication method and scope.
+     * The clients will be converted to OAuth2 ClientRegistration, the other information inherits from the delegated OAuth2 login client 'azure'.
+     * For instance, 'webapi:
+     *                   authorization-grant-type: on_behalf_of
```

Comment:
The indent will disappear when converted to `spring-configuration-metadata.json`